app/jobs/appointment_reminder_job.rb

```diff
@@ -5,7 +5,7 @@ def perform(appointment)
     ReminderActivity.create!(
       appointment: appointment,
       owner: appointment.guider,
-      message: 'sent a reminder'
+      message: 'sent an appointment reminder email to the customer'
```

Comment:
Absolutely. It scares me that some of this is hardcoded and some stored in the DB.